x-pack/plugins/fleet/server/services/epm/elasticsearch/template/template.ts

```diff
@@ -427,8 +430,8 @@ function _generateMappings(
             matchingType = field.object_type_mapping_type ?? 'object';
             break;
           default:
-            throw new Error(
-              `no dynamic mapping generated for field ${path} of type ${field.object_type}`
+            throw new FleetError(
+              `No dynamic mapping generated for field ${path} of type ${field.object_type}`
             );
```

Comment:
:+1: if this error happens is because of some invalid mapping in the package, so maybe we can throw an error about an invalid package.

This error can also happen if we are not handling some type that should be allowed here, but this would be a bug :slightly_smiling_face: